platform/client-admin/src/pages/ViewVolunteerPage.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { RouteComponentProps } from 'react-router-dom'
+import { Header } from '../components/Header';
+import { IUpdateMatchParams } from '../declarations';
+import { useUpdateVolunteerMutation, useFindVolunteersQuery, useFindVolunteersLazyQuery } from '../dataFacade'
+import { AutoForm } from 'uniforms-ionic'
+import volunteerFormSchema from '../forms/volunteer';
+import { volunteerTransformer } from '../transformer/volunteerTransformer';
+import { IonLoading, IonContent, IonList, IonCard, IonItemGroup, IonItemDivider } from '@ionic/react';
+
+export const ViewVolunteerPage: React.FC<RouteComponentProps<IUpdateMatchParams>> = ({ match }) => {
+  const { data, loading, error } = useFindVolunteersQuery({ fetchPolicy: "network-only", variables: { fields: {id: match.params.id}, limit: 1 } });
+  
+  const [updateVolunteer] = useUpdateVolunteerMutation();
+  if (error) {
+    console.log(error);
+  }
+
+  if (!data || !data.findVolunteers[0]) {
+    return <div>Cannot fetch element with provided id</div>
+  }
+
+  if (loading) return <IonLoading isOpen={loading} message={'Loading...'} />;
+  const model = volunteerTransformer(data.findVolunteers[0]);
```

Comment:
This is necessary to transform date to `Date` type also to clean up some unnecessary data. Is there a way to hook it into  the `useFindVolunteersQuery()` hook, i.e at linehttps://github.com/aerogear/OpenVolunteerPlatform/pull/35/files#diff-1989084bd5b246cffc56abd69d945556R12 I'll get the data I need in correct type ? @wtrocki

Reply:
It was comming as string when I was checking it last time as despite using DateTime scalar. Maybe we can avoid that and also remove this tranformation from the client.